wallet/bip32.go

```diff
@@ -109,7 +110,7 @@ func pubkeyFromLedger(path HDPath, master bool) (*EDKeyPair, error) {
 	// the one they really care about, which is the first child key.
 	key, err := ledger.ReadPubkeyFromLedger("", HDPathToString(path), !master)
 	if err != nil {
-		return nil, fmt.Errorf("error reading pubkey from ledger. Are you sure it's connected, unlocked, and the Spacemesh app is open? err: %w", err)
```

Comment:
It looks like we lost this line, @fasmat was this intentional?

Reply:
Yes, the linter complained about the overly long error message (errors aren't logs). I moved the message up the stack and log it there in case of an error.